scripts/superdesk-archive/views/upload.html

```diff
@@ -43,6 +43,11 @@ <h3 class="pull-left">{{ activity.label|translate }}</h3>
                         <div class="row"><input type="text" placeholder="{{ :: 'Add a slugline' | translate }} *" ng-model="item.meta.slugline"></div>
                         <div class="row"><input type="text" placeholder="{{ :: 'Add author / photographer' | translate }}" ng-model="item.meta.byline"></div>
                         <div class="row">
+                            <input type="text"
+                                   placeholder="{{ :: 'Who is claiming the property' | translate }}"
```

Comment:
This line was too long for me. When I have to scroll horizontally I prefer to cut